cluster-controller/src/main/java/io/strimzi/controller/cluster/operations/OperationExecutor.java

```diff
@@ -32,10 +34,18 @@ private OperationExecutor(Vertx vertx, K8SUtils k8s) {
         this.k8s = k8s;
     }
 
-    public void execute(Operation op, Handler<AsyncResult<Void>> handler) {
+    public void executeK8s(Operation<K8SUtils> op, Handler<AsyncResult<Void>> handler) {
         op.execute(vertx, k8s, handler);
     }
 
+    public void executeOpenShift(Operation<OpenShiftUtils> op, Handler<AsyncResult<Void>> handler) {
```

Comment:
Why do we keep the utils for the OpenShift but use Fabric8 for Kubernetes? Is that still WIP?